app.yaml

```diff
@@ -3,12 +3,22 @@ threadsafe: true
 api_version: 1
 
 env_variables:
-  CLOUDSQL_CONNECTION_NAME: 't35db24833ba76815-tp:us-central1:us-central1-istio-release-e-79a693d8-sql'
-  CLOUDSQL_HOST: '35.193.234.53'
-  CLOUDSQL_USER: 'root'
-  CLOUDSQL_PASSWORD: ''
-  CLOUDSQL_DB: 'airflow-db'
-  GCS_LOGS_BUCKET: 'us-central1-istio-release-e-79a693d8-bucket'
+  DB_NUM: 2
+
+  CLOUDSQL_CONNECTION_NAME_1: 't35db24833ba76815-tp:us-central1:us-central1-istio-release-e-79a693d8-sql'
+  CLOUDSQL_HOST_1: '35.193.234.53'
+  CLOUDSQL_USER_1: 'root'
```

Comment:
I think that one can stay as one global. since its root nothing anyway. I think you can also figure out the db name from the gcs bucket name or vice versa. that way your comma separated list becomes simple and then the parseAirflowConfig does all the work.